controller/userController.js

```diff
@@ -0,0 +1,272 @@
+const data = require("../model/user");
```

Comment:
variable naming has to be done , so that variable name defines what the variable is